terraform/app.tf

```diff
@@ -0,0 +1,44 @@
+resource "random_uuid" "id" {
+}
+locals {
+  secret_key_base = {
+    SECRET_KEY_BASE = "${random_uuid.id.result}"
```

Comment:
Just to check: this won't change SECRET_KEY_BASE between deployments right? Just once on the initial deploy.

Reply:
it will change each time it deploys. Do you want one constant value for each environment?

Reply:
Yeah. We need a constant value, because otherwise cookies will stop working between each deploy, and log users out.